getaround_utils/lib/getaround_utils/railties/lograge.rb

```diff
@@ -19,6 +19,11 @@ def append_info_to_payload(payload)
       payload[:lograge][:session_id] = session.is_a?(Hash) ? session[:id] : session.id.to_s if defined?(session)
       payload[:lograge][:user_id] = current_user&.id if defined?(current_user)
       payload[:lograge][:origin] = 'lograge'
+
+      return unless defined?(Newrelic::Agent::Tracer)
```

Comment:
Interesting choice to have this as a guard-close instead of a conditional-block.
Not that I'm against it, but I'm curious about the motivation (is it to save the indentation ?)

Reply:
The motivation is mostly because of Rubocop configuration in this repo. But I don't have a strong opinion about it.